aematei/newsly
# Install and configure nativewind for styling

Test:

1. In terminal, run `npx expo start --clear`
2. When expo is running, check that the app looks correct on your chosen platforms:
   - press 'i' to run on iOS (if you can)
   - press 'a' to run on Android
   - press 'w' to run on Web

The app should look like this:

![image](https://github.com/user-attachments/assets/94c22b3e-47c9-46da-9acb-71712ac109ba)
